Example/loudspeaker/LSPViewController.m

```diff
@@ -46,100 +55,55 @@ - (void)configureDemo:(id)sender
 {
     if (sender == self.harlequinButton)
     {
-        self.audioVC.view.progressView.foregroundColor = UIColor.redColor;
-        self.audioVC.view.progressView.backgroundColor = UIColor.blueColor;
-        self.audioVC.view.backgroundColor = UIColor.greenColor;
-        self.audioVC.view.playbackTimeLabel.textColor = UIColor.brownColor;
-        self.audioVC.view.titleLabel.textColor = UIColor.yellowColor;
+        self.configuration = [LSPConfigurationBuilder configurationWithBuilder:^(LSPConfigurationBuilder *builder) {
+            builder.height = 120;
```

Comment:
Why do we need set height in configuration? Why not let users specify player frame?

Reply:
Good point. Will change.